packages/core/test/controls/numericInputTests.tsx

```diff
@@ -57,6 +57,12 @@ describe("<NumericInput>", () => {
             const value = component.state().value;
             expect(value).to.equal("1");
         });
+
+        it("sets allowNumericCharactersOnly to true by default", () => {
+            const component = mount(<NumericInput />);
+            const value = component.props().allowNumericCharactersOnly;
+            expect(value).to.be.true;
```

Comment:
The test isn't here to test that React's `defaultProps` functionality works as expected; it's here to enforce the contract the component claims to uphold. During a refactor, if I accidentally delete or change a default value, that would affect consumers, and I'd want to know about it with a failing test.